src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsuofdataapi/integration/service/ReportServiceCoroutinesTest.kt

```diff
@@ -49,14 +49,14 @@ class ReportServiceCoroutinesTest {
       "user_id",
       1,
       1234,
-      LocalDateTime.now(),
+      LocalDateTime.of(2024, 1, 10, 14, 0),
```

Comment:
I have modified these DATES to have some consistent as .now it is more dynamic and they meant when it was created that incident, etc.